schema/spec.md

```diff
@@ -348,6 +357,25 @@ Versions for a schema named `com.example.telemetrydata`:
 }
 ```
 
+#### `validation`
+
+- Type: Boolean
+- Description: Indicates whether or not the server will validate the Resource's
+  document(s) against the specified `format` attribute. A value of `true`
+  indicates that the server MUST reject any request that would cause any
+  Version of this Resource to be invalid per the rules as defined by the
+  `format` specification. Note, this includes a request to set this attribute
+  to `true`.
+
+  A value of `false` indicates that the server MUST NOT do any validation.
+
+  If `format` is not specified, or if the value is not known by the server
+  (but it an allowable value), then the server MUST NOT perform any validation.
```

Comment:
"any" validation feels like it's potentially asking for trouble. Would it still be allowed to reject incoming JSON which specified the same property multiple times, for example? (That's technically-valid JSON, but feels reasonable to reject.) Presumably it's *always* okay to reject requests which don't have valid JSON, or are broken in other ways.

Basically I'm saying that we should limit the scope of "must not perform any validation" otherwise it sounds like "the server must not reject any request at all".

Reply:
The intent was to let any array of bytes in. Imagine cases where people are purposely uploading invalid docs, perhaps for sharing testing (invalid) docs or they're using a format that's only valid because of the tooling they're using (like some kind of new templating language for the format).  I see two ways for users to say "let anything in"... 1) turn off validation,  2) use a format that's the server doesn't know about and therefore can't check (which includes "no format").

Reply:
Right, thanks for the context. Presumably it's still only for the resource body though? I can't remember enough about other aspects of xRegistry requests to give concrete aspects, but I'd expect a sort of firewall between "the body of the resource, which is somewhat-opaque like CloudEvent data" and "resource metadata, authentication etc" which I'd still hope to be validated.

I'm hoping that you're able to clarify this in a more technically correct way, of course :)

Reply:
`Indicates whether or not the server will validate the Resource's document(s) against the specified `format` attribute`. It's the phrase "Resource's document" that's they key to me as that means it's that secondary doc associated with each Version that's checked and not the xReg metadata.  Think I need to elaborate more?

Reply:
That's fair enough - I suspect the nature of line-by-line PR review made me less sensitive to the context :)